docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -1,14 +1,65 @@
 # Development Environment
 
-The development environment setup guide is for setting up your local development environment for JavaScript development; creating plugins and tools for extending WordPress and the block editor.
+This guide is for setting up your local environment for JavaScript development for creating plugins and tools to extend WordPress and the block editor.
 
-A development environment is a catch-all term for the setup on your computer to work. The three main pieces needed for our development environment are:
+A development environment is a catch-all term for what you need setup on your computer to work. The three main pieces needed for our development environment are:
 
 1. Node/NPM Development Tools
 2. WordPress Development Site
 3. Code Editor
 
-## Development Tools
+## Quickstart
+
+For experienced developers, here is a summary of the guide, see each section for more details and explanations.
+
+**1. Install Node development tools**
+
+Download and install [Node Version Manager](https://github.com/nvm-sh/nvm) (nvm)
+
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+```
+
+Quit and restart terminal
+```
+nvm install --lts
+```
+
+**2. WordPress Development Site**
+
+First download, install, and start [Docker Desktop](https://www.docker.com/products/docker-desktop) follows instructions on their site for your OS.
+
```

Comment:
follows = and follow instructions for your OS.

There are some instructions to follow after starting up Docker for the first time. As one inserts the user password and a tutorial shows up which one can skip at any time.

It would be good with some info on why one is using Docker. What is it used for?

Reply:
This is in the quick start, I added a description below.